components/molecules/ContributorHighlight/contributor-highlight-card.tsx

```diff
@@ -657,12 +657,13 @@ const ContributorHighlightCard = ({
               direction="top"
               content={`${getEmojiReactors(reaction_users)} reacted with ${getEmojiNameById(emoji_id)} emoji`}
             >
-              <div
+              <button
                 className={`px-1 py-0 md:py-0.5 hover:bg-light-slate-6 transition  md:px-1.5 shrink-0 border flex items-center justify-center rounded-full cursor-pointer ${
                   isUserReaction(emoji_id) && "bg-light-slate-6"
                 }`}
                 onClick={async () => (sessionToken ? handleUpdateReaction(emoji_id) : signIn({ provider: "github" }))}
               >
+        </button>
```

Comment:
this button should go to line 671 and replace `</div` right after closing tag for `Emoji` component. I can't use `make a suggestion` button bc that line was added before the latest change.

Reply:
Thanks @Arol15! 😊 I'll try this tomorrow